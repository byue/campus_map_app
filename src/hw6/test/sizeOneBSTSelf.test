# Simple test of breadth-first search algorithm.
# Verifies output of BFS for size 1 graph.

CreateGraph characters
AddNode characters HAWK

FindPath characters HAWK HAWK