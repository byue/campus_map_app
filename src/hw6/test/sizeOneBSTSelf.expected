# Simple test of breadth-first search algorithm.
# Verifies output of BFS for size 1 graph.

created graph characters
added node HAWK to characters

path from HAWK to HAWK: